website/docs/expansion.md

```diff
@@ -110,24 +54,78 @@ spec:
       kinds: ["Job"]
       versions: ["v1"]
   templateSource: "spec.template"
-  enforcementAction: "warn"
+  enforcementAction: "warn" # this will overwrite all constraint enforcement actions for the GVKs below that result from the GVKs above
```

Comment:
```suggestion
  enforcementAction: "warn" # This will overwrite all constraint enforcement actions for the GVKs below that result from the GVKs above.
```